src/js/jsx/Search.jsx

```diff
@@ -86,6 +86,7 @@ define(function (require, exports, module) {
                             searchID={SEARCH_BAR_DIALOG_ID}
                             dismissDialog={this._closeSearchBar}
                             executeOption={this._handleOption}
+                            maxOptions={14}
```

Comment:
It will be easier to change this later if it's broken out into a `CONSTANT` with JSDoc.